.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -100,6 +101,9 @@ jobs:
     - script: |
         set "GIT_BRANCH=%BUILD_SOURCEBRANCHNAME%"
         set "FEEDSTOCK_NAME=%BUILD_REPOSITORY_NAME:*/=%"
+        set "TEMP=$(UPLOAD_TEMP)"
+        if not exist "%TEMP%\" md "%TEMP%"
+        set "TMP=%TEMP%"
```

Comment:
We recently came up with this workaround to release `conda` ( https://github.com/conda-forge/conda-feedstock/pull/189 ), which ran into a similar issue as this feedstock. Maybe it will work to address the issue seen here as well